### PR TITLE
Add Try Block to Lesson Events

### PR DESCRIPTION
**Pull request by michaelwehrley**
Add Try Block to Lesson Events

* Try block tests if lesson event has a header in position 2.

[Finished: #97251532]

### CODE REVIEW

File: app/views/think_feel_do_engine/learn/_completion_data.html.erb [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+    <td class="not-displayed">
+      <%= DateTime.iso8601(lesson_event[:lesson_selected_at]).to_i %>
+    </td>
+    <td class="not-displayed">
+      <%= DateTime.iso8601(lesson_event[:last_page_opened_at]).to_i %>
+    </td>
+    <td><%= ContentModules::LessonModule.find(lesson_event[:lesson_id]).title %></td>
+    <td>
+      <% lesson_event[:page_headers].try do |page_headers| %>
+        <%= page_headers[2] %>
```
Comment (michaelwehrley) [inline app/views/think_feel_do_engine/learn/_completion_data.html.erb]: @eschlange This is where the Big Bug Daddy lived! :bug:

**Comment by eschlange** [comment]
So, was this just an issue with staging data? Or is it possible for production data to have had the same issue?

**Comment by eschlange** [comment]
Looks good

**Comment by michaelwehrley** [comment]
No, I misread the error message and solved the wrong problem.  I then download the staging database and correctly diagnosed and solved the problem.

**Pull request merged**